MH4U Dex/ItemContainerViewController.m

```diff
@@ -57,11 +67,15 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
 {
     if ([segue.identifier isEqualToString:@"embedMonsterSources"]) {
         ItemSourcesTableViewController *dropVC = (ItemSourcesTableViewController *)segue.destinationViewController;
-        dropVC.isMonsterSource = YES;
+        dropVC.sourceType = @"Monster";
```

Comment:
Is it worth creating some constants for these sourceTypes to prevent accidental misspellings/miscapitalizations?